src/main/resources/data/minestuck/worldgen/processor_list/structure_block_registry.json

```diff
@@ -0,0 +1,7 @@
+{
+  "processors": [
+    {
+      "processor_type": "minestuck:block_entity"
```

Comment:
I wanted to ask, why was this called block_entity instead of something like block_registry?

Reply:
I can't imagine that it's anything but an accident. Go ahead and rename it.